Creating new salaried courses page (#3427)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/salaried-teacher-training.md

---
title: "Salaried teacher training"
description: |-
    Explore salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
related_content:
    Nathan's experience of salaried teacher training: "/blog/salaried-teacher-training-classroom-learning"
    If you've worked as an unqualified teacher: "/train-to-be-a-teacher/assessment-only-route-to-qts"
    What to expect on your teacher training: "/blog/what-to-expect-on-your-teacher-training"
promo_content:
    - content/train-to-be-a-teacher/promos/find-your-course
    - content/funding-and-support/promos/get-adviser-funding-promo
navigation: 20.20

> navigation_description: Find out about salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.

Reviewer: ```suggestion
navigation_description: Find out about salaried courses, including School Direct (salaried), postgraduate teaching apprenticeships and Teach First.
```

This is pretty long compared to our other nav descriptions - not sure whether my suggestion makes sense though!

Author: will tweak the first half, would probs be keen to keep the 'delivered by Teach First' bit though just to make sure we're 100% saying the right thing, policy wise

> navigation_description: Find out about salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.

Reviewer 2: Do we need to say 'salaried teacher training' as that's in the heading? Maybe something like:
```suggestion
navigation_description: Find out about School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
```

keywords:
    - Teach First
    - School Direct
    - School Direct (salaried)
    - Postgraduate teaching apprenticeship
    - PGTA
    - Salaried courses
    - Salaried teacher training
    - Salaried teacher training course
    - Paid teacher training
---

There are some teacher training courses that come with a salary. 

This means you will not have to pay tuition fees and will receive a salary while you train to get [qualified teacher status (QTS)](/what-is-qts). This is what you need to teach in many primary, secondary and special schools in England. 

The main salaried courses include School Direct (salaried), postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.

All salaried courses are broadly the same, but the course length and qualifications you work towards may vary.

> They're in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible. 

Reviewer: ```suggestion
These courses are in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible. 
```

(ignore if you disagree!)

## Am I eligible for a salaried teacher training course? 

For some salaried courses, teacher training providers may want you to have significant teaching or school experience (for example, if you’re already working as an unqualified teacher or teaching assistant).  

> Some also want you to have a school lined up to work in while you train. Speak to the teacher training provider to find out what their requirements are. 

Reviewer: Is there a better way of saying `lined up`? Particularly thinking of non-uk applicants - could we make this clearer?

```suggestion
Some also want you to have already arranged a school to work in while you train. Speak to the teacher training provider to find out what their requirements are. 
```

You can also [find out more about the qualifications you need to apply for all teacher training courses](/is-teaching-right-for-me/qualifications-you-need-to-teach). 

Or talk to a [teacher training adviser](/teacher-training-advisers) who can help you understand which courses you could be eligible for. 

## How much will I be paid on a salaried course? 

> You'll receive an [unqualified teacher’s salary](/salaries-and-benefits#unqualified-teacher-salaries) – the exact amount will vary depending on the school you work in. 

Reviewer: Normally we avoid links to anchors - however, I think this is probably appropriate here?

Reviewer: Just thought - the important information re pay award, which year this applies to etc. is at the top of the Salaries page. By linking to an anchor, we are not giving people a chance to see this information @EllieNodder ?

Author: good point - shall remove the anchor and just leave overall link in

> It’s worth checking if you’re entitled to a [bursary or scholarship](/funding-and-support/scholarships-and-bursaries) before you apply for a salaried course. 

Reviewer: Think this section might need to be reworded in some places @EllieNodder 

Whilst you might be entitled to a bursary, you'd still have to apply for a scholarship?

Author: be better to just say 'eligible for' rather than 'entitled to', then it applies to both?

Reviewer: ```suggestion
It’s worth checking if there is a [bursary or scholarship](/funding-and-support/scholarships-and-bursaries) available for your subject before you apply for a salaried course. 
```
How about something like this?

If you do a salaried course, you will not be eligible for a bursary, scholarship or student finance. You’ll also be taxed on your income, whereas you will not be taxed on a bursary or scholarship. 

> If you’re eligible for a bursary or scholarship, you could receive more money on a non-salaried course than on a salaried one. 

Author: @gemmadallmandfe @philspray feel like I'm saying 'scholarships and bursaries' a lot here, but want to make sure we're being clear, so happy to have any tweaks recommended here particularly!

Reviewer: I think it's fine as is!

Reviewer 2: Yep me too!

## How does salaried teacher training work? 

Salaried teacher training is usually similar to non-salaried teacher training – you'll spend most of your time in school placements, with some theoretical learning. 

You will not perform more than 90% of a full-time teacher’s duties (unless you’re on a postgraduate teaching apprenticeship, in which case it’s 80%). 

Full-time salaried teacher training will usually last: 

> * 1 academic year (September to July) if you’re on a School Direct (salaried) course 

Author: @gemmadallmandfe @philspray know we've talked about removing academic year, but I feel like in this case because we validate it with months, it's okay?

Reviewer 2: Tiny suggestion, probably makes no difference but would it be better the other way around? Putting the less familiar term in the brackets?

```suggestion
* from September to July (1 academic year) if you’re on a School Direct (salaried) course 
```

Reviewer: I think your suggestion is good @philspray where there is only one year involved but not sure it would work for the 2 year Teach First bullet?

In which case, I think it is probably OK to leave as is, given that we've explained the months involved in brackets?

* 2 academic years (September to July) if you’re on a course delivered by Teach First 

* at least 1 full year (September to September) if you’re on a postgraduate teaching apprenticeship – apprenticeships must last a minimum of 12 months 

Some courses can begin at other points in the year and there may be part-time courses available. 

As well as QTS, you may also be able to work towards a [postgraduate certificate in education (PGCE)](/what-is-a-pgce), but you may have to pay an extra fee. 

> [Find out more about what to expect as part of your teacher training](/train-to-be-a-teacher/initial-teacher-training). 

Reviewer: ```suggestion
[Find out more about what to expect during your teacher training](/train-to-be-a-teacher/initial-teacher-training). 
```
